Remove trailing slashes from internal links (#1456)
semgrep/semgrep-docs: src/components/reference/_ci-ignoring-files.mdx

`.semgrepignore` is only used by Semgrep. Integrations such as [GitLab's Semgrep SAST Analyzer](https://gitlab.com/gitlab-org/security-products/analyzers/semgrep) do not use it.
:::

> For information on ignoring individual findings in code, see the [Ignoring findings page](/ignoring-findings).

Reviewer: ```suggestion
For information on ignoring individual findings in code, see [Ignoring files, folders, and code](/ignore-oss).
```
